pkg/arm/resources.go

```diff
@@ -591,15 +591,17 @@ func Vmss(pc *api.PluginConfig, cs *api.OpenShiftManagedCluster, app *api.AgentP
 		Location: to.StringPtr(cs.Location),
 	}
 
+	// The additional data disk have different purposes on master
+	// and infra/compute nodes, but they are both the same size
```

Comment:
good comment - please add further rationale: we use 256GiB for etcd because of the iops; for docker we use it to match OSD (size)